Add vehicle page

Eswar-S-Sethu opened the pull request:
Changed the environment to React Native.

Eswar-S-Sethu committed:
added my contribution to the add vehicle page

Eswar-S-Sethu committed:
added my contribution to the add vehicle page

Eswar-S-Sethu committed:
added my contribution to the add vehicle page

Rahulseh committed:
Added the option section, the add vehicle year, and chat box icon with country section.

Eswar-S-Sethu committed:
completed the basic look of the Add Vehicle page

Eswar-S-Sethu committed:
completed the basic look of the Add Vehicle page

Eswar-S-Sethu committed:
completed the basic look of the Add Vehicle page

Eswar-S-Sethu committed:
completed the basic look of the Add Vehicle page

Eswar-S-Sethu committed:
added background

Eswar-S-Sethu committed:
made the necessary changes

Eswar-S-Sethu committed:
made the necessary changes

Eswar-S-Sethu committed:
adjusted the looks

Eswar-S-Sethu committed:
changed environment

Eswar-S-Sethu committed:
removed all files of the previous env

Eswar-S-Sethu committed:
test

Eswar-S-Sethu committed:
changed env

sthenusan commented:
@Eswar-S-Sethu ,  you have to resolve the conflicts in this in this PR before the review. and keep the number of commits minimal as possible.

sthenusan commented:
@Eswar-S-Sethu , Just notice that, your PR have some issues as it shows that 1090 files have changed in this PR, please use `.gitignore` file in efficient way to remove unnecessary files getting push to remote repo.

4DRI4NT committed:
Fix: Suggestion dropdown

App.tsx edited so suggestion dropdown now selects, fills text input, shows/hides
